### PR TITLE
weboob 1.4

### PR DESCRIPTION
**Pull request by laurentb**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Weboob 1.4 released: https://git.weboob.org/weboob/weboob/tags/1.4 https://pypi.org/project/weboob/1.4/#files

### CODE REVIEW

**Comment by fxcoudert** [comment]
Dependencies need to be updated (at least `certifi` and `six`, possibly many others) to the latest compatible version:

```
        'lxml',
        'cssselect',
        'requests>=2.0.0',
        'python-dateutil',
        'PyYAML',
        'html2text>=3.200',
        'six',
        'unidecode',
        'Pillow',
```

File: Formula/weboob.rb [hunk 1]
```diff
@@ -3,10 +3,9 @@ class Weboob < Formula
 
   desc "Web Outside of Browsers"
   homepage "https://weboob.org/"
-  url "https://git.weboob.org/weboob/weboob/uploads/e8c77b143e84c97fca354579575d18e9/weboob-1.3.tar.gz"
-  mirror "https://deb.debian.org/debian/pool/main/w/weboob/weboob_1.3.orig.tar.gz"
-  sha256 "c991785c889877c76f18d19e372ed4ae0c3f8b819fd1e8da296bd34b1381be54"
-  revision 2
+  url "https://git.weboob.org/weboob/weboob/uploads/08dad6b2a68fd618ba8c87c627393d06/weboob-1.4.tar.gz"
+  mirror "https://files.pythonhosted.org/packages/34/e7/230d679030e6970be3c98339811264999c5484ad657feb9a95e3f9ce5035/weboob-1.4+1.tar.gz"
```
Comment (Moisan) [inline Formula/weboob.rb]: Could you keep the mirror pointing to `deb.debian.org`?

**Review by Moisan — CHANGES_REQUESTED** [review]

**Comment by Moisan** [comment]
By updating dependencies, we mean to look at each `resource` block in the formula and making sure they are up to date with their latest release (on pypi in this case).

**Comment by laurentb** [comment]
I see, I didn't realize at first that dependencies were pinned to exact versions. I've also removed a dependency that shouldn't be there.

If you want I can squash the two commits.

File: Formula/weboob.rb (hunk 1 above)
Comment (laurentb) [reply Formula/weboob.rb]: Version 1.4 isn't available yet in the Debian mirrors, and it is usually very long to get an update pushed. I have an update pending for Gentoo too, if it gets merged I could switch to their mirrors if you prefer.

It's the first time we release a tarball properly (the same pushed on pypi and provided to other users) hence the change to pypi. The other advantage is that for new versions we can update it immediately.

**Comment by Moisan** [comment]
Yes that's a good practice to squash the commits. There is currently styling issue with the last commit:
```
C: 49:  3: Use 2 (not 4) spaces for indentation.
C: 50:  5: Inconsistent indentation detected.
```

**Comment by laurentb** [comment]
My editor wasn't configured for formulae styling so it kept trying to use 4 spaces and I fixed them manually... which was a bad idea as I missed one. Sorry!

Fixed & squashed.

**Review by Moisan — APPROVED** [review]

**Comment by Moisan** [comment]
The preferred commit message would have been `weboob 1.4` but it's okay I amended the message. Thank you for the PR @laurentb !

**Comment by laurentb** [comment]
Noted, next time I'll add the details in the message, not the subject

Thanks!

**Pull request closed without merging**